crates/ruff_workspace/src/options.rs

```diff
@@ -29,30 +29,6 @@ use strum::IntoEnumIterator;
 #[serde(deny_unknown_fields, rename_all = "kebab-case")]
 #[cfg_attr(feature = "schemars", derive(schemars::JsonSchema))]
 pub struct Options {
-    /// A list of allowed "confusable" Unicode characters to ignore when
-    /// enforcing `RUF001`, `RUF002`, and `RUF003`.
-    #[option(
```

Comment:
The documentation moves to the new option names but we also remove the old settings from the schema. Is this fine?

Reply:
How does this manifest in the documentation? Trying to reconcile this with the PR summary (and also grok the code). Like what does https://docs.astral.sh/ruff/settings look like now?

Reply:
They are unchanged because the new lint field isn't attributed with option_group

We could change that but I think it would make it too verbose. The only real alternative IMO is to add alias support

Reply:
This end-result seems right to me.